br-webui/views/network.liquid

```diff
@@ -226,31 +226,31 @@ socket.on("current network config", function(data) {
 // Set companion network configuration according to the option selected by user
 function setCompanionMode() {
 	if (document.getElementById("manual").checked) {
-		if (confirm("Are you sure you want to configure Companion \
-as the default Manual mode? The system will automatically reboot for \
-reconfiguration")) {
+		if (confirm("Are you sure you want to configure the Companion Ethernet interface in Manual mode? The system\
+		will automatically reboot for reconfiguration. The static IP address will be 192.168.2.2 after reboot.\
+		CLICKING OK MAY RENDER THE SYSTEM UNUSABLE. IF YOU ARE NOT SURE OF WHAT YOU ARE DOING, CLICK CANCEL!")) {
```

Comment:
This extra spaces in the string will make it weird for the user.
![screenshot from 2018-12-05 09-07-36](https://user-images.githubusercontent.com/1215497/49509836-a5b49880-f86d-11e8-9fdd-a0159237f134.png)

Reply:
It's already been fixed. Check your commit.

Reply:
Yep:
![screenshot from 2018-12-05 14-30-54](https://user-images.githubusercontent.com/1215497/49528275-cfd07f80-f89a-11e8-88ca-ec5d6cff8245.png)

You can also test this issue with google chrome console.
![screenshot from 2018-12-05 14-32-25](https://user-images.githubusercontent.com/1215497/49528416-0a3a1c80-f89b-11e8-97c0-45fd0f9de5e9.png)

Reply:
@patrickelectric done. The expectation was the html rendered doesn't show tabs/newlines.